graphql/schema/types/project_subscriber.graphql

```diff
@@ -51,6 +54,9 @@ input WebhookSubscriberInput {
   headers: [WebhookHeaderInput]!
   secret: String!
   url: String!
+  retries: Int = 0
```

Comment:
Should these correspond to the defaults?

Reply:
I think at least for right now, it's ok to set them to 0. There are a bunch of subscriptions in the production DB with these values set to 0 (and it might be confusing for that pattern to suddenly end). I double checked with Jonathan and he said that if these values are left at 0 in the database, they will get set to the defaults when sending the webhook.